src/Instrumentation/ExtAmqp/src/ExtAmqpInstrumentation.php

```diff
@@ -0,0 +1,225 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Contrib\Instrumentation\ExtAmqp;
+
+use AMQPExchange;
+use AMQPQueue;
+use OpenTelemetry\API\Globals;
+use OpenTelemetry\API\Instrumentation\CachedInstrumentation;
+use OpenTelemetry\API\Trace\Span;
+use OpenTelemetry\API\Trace\SpanKind;
+use OpenTelemetry\API\Trace\StatusCode;
+use OpenTelemetry\Context\Context;
+use OpenTelemetry\Context\Propagation\ArrayAccessGetterSetter;
+use function OpenTelemetry\Instrumentation\hook;
+use OpenTelemetry\SemConv\TraceAttributes;
+use Throwable;
+
+final class ExtAmqpInstrumentation
+{
+    public const NAME = 'ext_amqp';
+
+    public static function register(): void
+    {
+        $instrumentation = new CachedInstrumentation('io.opentelemetry.contrib.php.ext_amqp');
```

Comment:
Should add version and schema url.

Reply:
I've set the version by reflecting through the composer runtime API, there is no schema URL to my knowledge

Reply:
I think we don't do this anywhere, but should: https://github.com/open-telemetry/opentelemetry-specification/blob/v1.27.0/specification/schemas/README.md

> Third-party libraries, instrumentation or applications are advised to define and publish their own telemetry schema if it is completely different from OpenTelemetry schema (or use OpenTelemetry schema) and include the schema URL in the emitted telemetry.

Linked from there is an example of how go does it, and they use the semconv schema url, so `ResourceAttribute::SCHEMA_URL` seems like the right value to use.

Reply:
Okay, pushed